web/html/xui/form/protocol/trojan.html

```diff
@@ -1,12 +1,6 @@
 {{define "form/trojan"}}
 
-
-
-
 <a-form layout="inline">
-    <a-form-item label='{{ i18n "password" }}'>
```

Comment:
1- are you deleting password from trojan? 
2-  trojan doesn't have any id field!
3-inbound.settings.clients[0].flow should be in each client no just first one

Reply:
hi, I added the password field for each client also I added xtls flow to each client inside add client inbox and I didn't quite understand the id field you mentioned there is an id field in add client for each client is there something I missing?

Reply:
I mean as you see here https://github.com/hossinasaadi/x-ui/blob/main/web/html/xui/form/protocol/trojan.html
the trojan protocol just has a password and flow! not uuid(id) and alterID field.

Reply:
https://github.com/hossinasaadi/x-ui/blob/ba6d4e7a6d03c8427ed0ccae587354c0ebba57d0/web/assets/js/model/xray.js#L1388
this is base trojan client and you actually make mistake here, just compare it with your files.